runtime/config.go

```diff
@@ -58,6 +58,9 @@ type Config struct {
 // TODO: Add netns field
 type JailerConfig struct {
 	RuncBinaryPath string `json:"runc_binary_path"`
+	// JailPath is used to determine where the jailed application should be
+	// contained in.
```

Comment:
This comment should be a bit more precise. "Jailing" is not necessarily a well defined term, so we should be really clear about what we mean when we talk about it. Something like "JailPath specifies the location of a directory contain per-VM minimal root directories for use in private mount namespaces".